.github/workflows/multi-approvers.js

```diff
@@ -2,49 +2,120 @@ const APPROVED = 'APPROVED';
 const COMMENTED = 'COMMENTED';
 const MIN_APPROVED_COUNT = 2;
 
-/** Returns the number of approvals from members in the given list. */
-function inOrgApprovedCount(members, submittedReviews, prLogin) {
-  const reviewStateByLogin = {};
-  submittedReviews
-    // Remove the PR user.
-    .filter((r) => r.user.login !== prLogin)
-    // Only consider users in the org.
-    .filter((r) => members.has(r.user.login))
-    // Sort chronologically ascending. Note that a reviewer can submit multiple reviews.
-    .sort((a, b) => new Date(a.submitted_at) - new Date(b.submitted_at))
-    .forEach((r) => {
-      const reviewerLogin = r.user.login;
-
-      // Set state if it does not exist.
-      if (!Object.hasOwn(reviewStateByLogin, reviewerLogin)) {
-        reviewStateByLogin[reviewerLogin] = r.state;
-        return;
-      }
+/** Implements approvedCount using the contains method. */
+class AbstractMembers {
+
```

Comment:
We don't really write a lot of js so we don't have a linter yet, but there shouldn't be an extra newline here.

Reply:
Will address this in a different PR where I will use an Action instead of a Workflow.

Reply:
Please create a follow-up issue.